components/bl-mapbox/component.json

```diff
@@ -1,5 +1,5 @@
 {
-  "id": "c_8f971f6aef45e2a67d942f97229a378a",
```

Comment:
Is it OK that id was changed?

Reply:
Just created a copy of the component to separate tickets. Could it be a problem after republishing?

Reply:
You should ask Stas, but I think yes